docs/install.rst

```diff
@@ -345,6 +345,8 @@ packages:
 
     - `Sphinx <http://sphinx.pocoo.org>`_ (and its dependencies) 1.0 or later
```

Comment:
the sphinx dependency is now indirect and comes from `sphinx-astropy` doesn't it?

Reply:
Yes but I think we should still mention Sphinx as a dependency explicitly given that it's a pretty important one, just so we have an excuse to link to it?

Reply:
Yes, keep listing it, but make explicit that it is pulled in by `sphinx-astropy` (as is `Graphviz`, no? or is that used for anything other than documentation?) So, I would just add a parenthetical sentence ... "1.0 or later (this in turn depends on ... sphinx ... graphviz)"